providers/pool.rb

```diff
@@ -120,120 +120,150 @@ def site_identifier
+    
+    # recycling items
+    configure_application_pool(is_new_log_event_on_recycle, "recycling.logEventOnRecycle:PrivateMemory,Memory,Schedule,Requests,Time,ConfigChange,OnDemand,IsapiUnhealthy")
+    configure_application_pool(@new_resource.private_mem && is_new_private_memory, "recycling.periodicRestart.privateMemory:#{@new_resource.private_mem}")
+    configure_application_pool(is_new_disallow_rotation_on_config_change, "recycling.disallowRotationOnConfigChange:#{@new_resource.disallow_rotation_on_config_change}")
+    configure_application_pool(is_new_disallow_overlapping_rotation, "recycling.disallowOverlappingRotation:#{@new_resource.disallow_overlapping_rotation}")
+
+    # failure items
+    configure_application_pool(is_new_load_balancer_capabilities, "failure.loadBalancerCapabilities:#{@new_resource.load_balancer_capabilities}")
+    configure_application_pool(is_new_orphan_worker_process, "failure.orphanWorkerProcess:#{@new_resource.orphan_worker_process}")
+    configure_application_pool(is_new_orphan_action_exe, "failure.orphanActionExe:#{@new_resource.orphan_action_exe}")
+    configure_application_pool(is_new_orphan_action_params, "failure.orphanActionParams:#{@new_resource.orphan_action_params}")
+    configure_application_pool(is_new_rapid_fail_protection, "failure.rapidFailProtection:#{@new_resource.rapid_fail_protection}")
+    configure_application_pool(is_new_rapid_fail_protection_interval, "failure.rapidFailProtectionInterval:#{@new_resource.rapid_fail_protection_interval}")
+    configure_application_pool(is_new_rapid_fail_protection_max, "failure.rapidFailProtectionMaxCrashes:#{@new_resource.rapid_fail_protection_max_crashes}")
+    configure_application_pool(is_new_auto_shutdown_exe, "failure.autoShutdownExe:#{@new_resource.auto_shutdown_exe}")
+    configure_application_pool(is_new_auto_shutdown_params, "failure.autoShutdownParams:#{@new_resource.auto_shutdown_params}")
+
+    # cpu items
+    configure_application_pool(is_new_cpu_action, "cpu.action:#{@new_resource.cpu_action}")
+    configure_application_pool(is_new_cpu_limit, "cpu.limit:#{@new_resource.cpu_limit}")
+    configure_application_pool(is_new_cpu_reset_interval, "cpu.resetInterval:#{@new_resource.cpu_reset_interval}")
+    configure_application_pool(is_new_cpu_smp_affinitized, "cpu.smpAffinitized:#{@new_resource.cpu_smp_affinitized}")
+    configure_application_pool(is_new_smp_processor_affinity_mask, "cpu.smpProcessorAffinityMask:#{@new_resource.smp_processor_affinity_mask}")
+    configure_application_pool(is_new_smp_processor_affinity_mask_2, "cpu.smpProcessorAffinityMask2:#{@new_resource.smp_processor_affinity_mask_2}")
+
+    # Application Pool Identity Settings
     if ((@new_resource.pool_username && @new_resource.pool_username != '') and
       (@new_resource.pool_password && @new_resource.pool_password != '') and
       !is_new_user_name and
       !is_new_password)
```

Comment:
I thought there was a commit against libraries/helper.rb as part of #107 that flipped the working on is_new.. to return it's actual value instead of the opposite value.  I think we need to go in and actually rethink some of our logic here.  I totally understand why you'd want to not change too many things as part of this pull request (especially since we don't have a working test environment) but another look into this after we do might be warranted.